active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+_Behavior update to existing API when running as Desktop app_
+
+Invoked when the application is launched. Override this method to perform application initialization 
+and to display initial content in the associated Window (UWP apps) or create a new Window (Desktop apps).
+
+```CS
+protected virtual void OnLaunched(XamlLaunchActivatedEventArgs args)
+```
+
+> This method is typically created automatically when a project is created. 
+You can then modify it or remove it.
+
+### Examples
+
+In a Desktop app, use OnLaunched to create the main window and pass it the first command line argument.
+
+```CS
+protected override void OnLaunched(XamlLaunchActivatedEventArgs e)
+{
+     MainWindow mainWindow = new MainWindow();
+     
+     if(e.Arguments != null)
+     {
+        string[] arguments = e.Arguments.Split(' ');
+        // Assuming MainWindow has a public property called FileName 
+        mainWindow.FileName = arguments[0];
+     }	
+
+     mainWindow.Activate();
+}
+```
+
+In a UWP app,
+
+```CS
+protected override void OnLaunched(XamlLaunchActivatedEventArgs e)
+{
+     MainPage page = new MainPage();
+     
+     if(e.Arguments != null)
+     {
+        string[] arguments = e.Arguments.Split(' ');
+	    // Assuming MainPage has a public property called FileName 
+        page.FileName = arguments[0];
+     }	
+
+    Window.Current.Content = page;
+    Window.Current.Activate();
+    window.Activate();
+}
+```
+
+
+## Application.RequiresPointerMode property
+_Behavior update to existing API when running as Desktop app_
+
+Gets or sets whether a UWP app supports mouse mode, which emulates pointer interaction experiences 
+with non-pointer input devices such as an Xbox gamepad or remote control. (All nested elements inherit 
+this behavior.)
+
+> Note: this property is ignored in a Desktop app
+
+## Application.OnWindowCreated(XamlWindowCreatedEventArgs) method
+_Behavior update to existing API when running as Desktop app_
+
+Invoked when the application creates a window.
+
+```CS
+protected virtual void OnWindowCreated(XamlWindowCreatedEventArgs args)
+```
+
+### Remarks
+- In a UWP app, this method is called once for the creation of the main window, once for each hosted window, 
+and once each time CreateNewView is called. 
+- In a Desktop app, this method is called every time a Window object is created.
+
+## XamlWindowActivatedEventArgs class
+
+Contains the windows activation state information returned by the Window.Activated event.
+
+> Issue: should rename these to avoid collision.
+
+> Spec note: this is a new XAML version of the existing 
+Windows.UI.Core.[WindowActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.UI.Core.WindowActivatedEventArgs).
+
+## XamlWindowActivatedEventArgs properties
+
+**Handled**
+
+Gets or sets whether the window activation changed event was handled.
+ **true** if the event has been handled by the appropriate delegate; **false** if it has not.
```

Comment:
This is an "-ed" event, so the activation has already occurred. The Handled would presumably merely prevent other event handlers from seeing the event? That seems like a weird thing to do. What's the use case for handling the event?